edu-services/cm-service/cm-impl/hibernate-impl/hibernate/src/java/org/sakaiproject/coursemanagement/impl/CourseOfferingCmImpl.java

```diff
@@ -138,4 +138,20 @@ public String getStatus() {
 	public void setStatus(String status) {
 		this.status = status;
 	}
+
+	@Override
+	public String getTitle() {
+		if (title == null || title.equals(""))
+			return canonicalCourse.getTitle();
+
+		return title;
+	}
+
+	@Override
+	public String getDescription() {
+		if (description == null || description.equals(""))
```

Comment:
description == null || description.equals("") can be replaced with StringUtils.isEmpty(), which is null safe